vue/test/components/scrollable-items/scrollable-items.test.js

```diff
@@ -0,0 +1,394 @@
+const assert = require("assert");
+const base = require("../../base");
+
+describe("Scrollable Items", () => {
+    it("should initiate the component", async () => {
+        const component = base.getComponent("ScrollableItems", {
+            props: {
+                items: [
+                    { label: "Item Example 1", value: "item1" },
+                    { label: "Item Example 2", value: "item2" },
+                    { label: "Item Example 3", value: "item3" },
+                    { label: "Item Example 4", value: "item4" },
+                    { label: "Item Example 5", value: "item5" },
+                    { label: "Item Example 6", value: "item6" }
+                ]
+            }
+        });
+
+        // Setting "scrollableWidth" greater than 0 is a workaround to make the test work properly.
+        // It makes the component think it doesn't have enough space to show all the items, thus
+        // the default behaviour will be to show the arrows
+        await component.setData({ scrollableWidth: 500 });

```

Comment:
Ideally we should try and change the size of the window instead. Besides being closer to real usage, and not relying on internal implementation details, it can later be re-used in other tests.

Try something like:

```javascript
function resizeWindow({ width: null, height: null } = {}) {
    window.innerWidth = width || window.innerWidth;
    window.innerHeight = height || window.innerHeight;
    window.dispatchEvent(new Event("resize"));
}
```

Reply:
@gcandal Unfortunately we can't do this, because jsdom doesn't implement a CSS layout box, so atributes like an element `scrollLeftMax` or `clientWidth` aren't computed, so this will trigger the resize event but we can't compute elements sizes changes.

Resources:
- How atributes like clientWidth should behave when no CSS layout box is used: https://www.w3.org/TR/cssom-view/#dom-element-clientwidth
- Explanation of why zero is returned for atributes like clientWidth: https://github.com/jsdom/jsdom/issues/2310
- Discussion of the implementation of a CSS Layout box: https://github.com/jsdom/jsdom/issues/1322

Reply:
Great investigation there 👍 

Please move this mocking to a separate function. While jsdoc'ing it, add all this as notes there